Client/FeatureFlags/NimbusFlaggableFeature.swift

```diff
@@ -28,6 +28,7 @@ enum NimbusFeatureFlagID: String, CaseIterable {
     case pullToRefresh
     case recentlySaved
     case reportSiteIssue
+    case rustSyncManagerComponent
```

Comment:
Since James approved how Nimbus is being called in `AppDelegate`, I hesitate to change that so I think this can be deleted. I was going to push up the changes below but there a lot of conflicts present in merge the remote epic branch with my local one.

Reply:
@nbhasin2 I created #13770 since I don't have write access to this branch. Let me know if there's a better way to push these changes.